spec/npmdc_spec.rb

```diff
@@ -195,4 +195,18 @@
       expect { subject }.to write_output(output_msg)
     end
   end
+
+  context 'success scoped packages check' do
```

Comment:
Hm, this line https://github.com/lysyi3m/npmdc/pull/25/files#diff-973e9dcb9c84bebcd9d56b7b71dbbae8R72.

We should check that it doesn't raise an error if `package.json` is missing.